### DISCUSSION TITLE
SQLログ出力の制御

### DISCUSSION THREAD

**Discussion by gikeihi** [domaframework/doma#795] [Q&A] [answered]
一部SQLのログ出力を排除する方法がありますか？
開発環境とステージング環境で、SQLのログを出力していますが、一部SQLを排除したいです。
例えば、１秒一回実施するスケジュール処理は、大量のSQLログを出力してしまいます。

**Comment by nakamura-to** [comment, marked as answer]
DomaそのものにはSQLログ出力を制御する機能はありません。
Logライブラリで実現するのはいかがですか？
例えばLogbackであれば次のクラスが使えそうです。
https://logback.qos.ch/apidocs/ch/qos/logback/classic/turbo/DynamicThresholdFilter.html

DomaとLogbackを連携させるにはdoma-slf4jを使ってください。下記のドキュメントが参考になると思います。
https://doma.readthedocs.io/en/latest/slf4j-support/

**Reply by gikeihi** [reply]
ご回答、ありがとうございました。
`DomaそのものにはSQLログ出力を制御する機能はありません。`
承知いたしました。
実現は、LogbackのTurboFilterを検討いたします。
